### PR TITLE
Fixes #24 - 2nd pot/prepot

### LINKED ISSUE #24: General: Add castefficiency for prepot/second pot
As a healer, check that potion of prolonged, potion of deadly grace or potion of old war was active on_initialize.

Also check that potion of prolonged, potion of deadly grace, potion of old war, leytorrent potion or ancient mana potion has been used in fight.

Otherwise add as suggestions.

### PR DESCRIPTION
**Pull request by buimichael**

### CODE REVIEW

File: src/Parser/Core/CombatLogParser.js [hunk 1]
```diff
@@ -249,6 +251,29 @@ class CombatLogParser {
         });
       }
     }
+    if (!this.modules.prePotion.usedPrePotion) {
+      results.addIssue({
+        issue: <span>You forgot to use a pre potion before a pull. Using a prepotion can be very effective (even for healers), especially during shorter encounters.</span>,
```
Comment (MartijnHols) [inline src/Parser/Core/CombatLogParser.js]: More basic info might help some users, e.g.;
`You forgot to use a potion before combat. Using a potion before combat allows you the benefit of two potions in a single fight. A potion such as <ItemLink id={ITEMS.POTION_OF_PROLONGED_POWER.id} /> can be very effective (even for healers), especially during shorter encounters.`

File: src/Parser/Core/CombatLogParser.js [hunk 2]
```diff
@@ -249,6 +251,29 @@ class CombatLogParser {
...
+        icon: ITEMS.POTION_OF_PROLONGED_POWER.icon,
+        importance: SUGGESTION_IMPORTANCE.MINOR,
+      });
+    }
+    if (!this.modules.prePotion.usedSecondPotion) {
+      let importance;
+      let issue;
+      if(!this.modules.prePotion.neededManaSecondPotion) {
+        importance = SUGGESTION_IMPORTANCE.MINOR;
+        issue = <span>You forgot to use a second potion during the encounter. Even if you didn't need a mana potion, a potion of prolonged always helps your throughput.</span>;
```
Comment (MartijnHols) [inline src/Parser/Core/CombatLogParser.js]: This can be a string

Comment (buimichael) [inline src/Parser/Core/CombatLogParser.js]: Not sure how this is done

**Comment by MartijnHols** [comment]
Thanks!

**Pull request merged**